packages/runtime-common/realm.ts

```diff
@@ -250,6 +252,7 @@ export class Realm {
     let localPath = this.paths.local(url);
     if (
       accept?.includes('application/vnd.api+json') ||
+      accept?.includes(cardJsonMimeType) ||
```

Comment:
this is problematic. right now we have a json api router that handles both directory requests and card instance requests.  I think we need a card json router that handles the card requests and a json api router that handles the directory requests. this way we can add a route for the index card in the card json router which won't collide with the directory route in the jsonapi router. The idea is that accept headers that are json-api are sent to the json api router, and accept headers that are card json are send to the card json router.

Reply:
BTW, an interesting enhancement to our routers would be the ability to add accept header considerations to them, i think we could write more declarative routing rules that way....